decode/neuralfitter/train/live_engine.py

```diff
@@ -269,7 +270,7 @@ def setup_trainer(simulator_train, simulator_test, logger, model_out, ckpt_path,
     """Log the model"""
     try:
         dummy = torch.rand((2, param.HyperParameter.channels_in,
-                            *param.Simulation.img_size), requires_grad=True).to(torch.device(device))
+                            *param.Simulation.img_size), requires_grad=False).to(torch.device(device))
```

Comment:
Does this really log the graph? If requires_grad is False? Have you checked in tensorboard?

Reply:
Yes, it still works. Though one could argue about the use usefulness of the graph.